src/wallet/rpcdump.cpp

```diff
@@ -573,7 +582,10 @@ UniValue importwallet(const JSONRPCRequest& request)
                 },
             }.ToString());
 
-    if (pwallet->chain().getPruneMode()) {
+    if (pwallet->chain().havePruned()) {
```

Comment:
faf3729242c5e9486e137e549f05bf20bd3908a0

The dump file contains timestamps so, correct me if I'm wrong, it could only throw this error if earliest is pruned - the same for `importmulti` if timestamps are given.

Reply:
Sounds like a nice follow up pull request